src/main/java/org/openjdk/jextract/JextractTool.java

```diff
@@ -360,8 +362,8 @@ private int run(String[] args) {
         parser.accepts("--output", "help.output", true);
         parser.accepts("-t", List.of("--target-package"), "help.t", true);
         parser.accepts("--version", "help.version", false);
-        parser.accepts("--macos-framework", "help.mac.framework", true);
-        parser.accepts("-f", "help.framework.library.path", true);
+        parser.accepts("-framework", "help.mac.framework", true);
+        parser.accepts("-F", "help.framework.library.path", true);
```

Comment:
I think these should be the other way around. `-F` specifies the search path, and `-framework` accepts the name of the framework you want to link against (e.g. `-framework OpenGL`).